save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/DemoRequest.kt

```diff
@@ -0,0 +1,6 @@
+package com.saveourtool.save.demo
+
+/**
+ * Interface that should be implemented by all data classes that are meant to be some demo service run request.
+ */
+interface DemoRequest
```

Comment:
Is an empty parent interface needed now if it doesn't add any common logic?

Reply:
Yes, as far as we require it as an argument. Require Any seems to be a bad practice. Moreover, if we want to require more, we will simply do that